docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -756,6 +757,20 @@ Therefore, authenticated users can access the secured application area without r
 Default resolution can also work for Bearer token authentication.
 Still, it might be less practical because a tenant identifier must always be set as the last path segment value.
 
+[[issuer-based-tenant-resolver]]
+=== Resolve tenants with issuer
+
+Quarkus supports static tenant selection based on the Bearer token issuer.
+For this to work, you need to configure your application like in the example below:
+
+[source,properties]
+----
+quarkus.oidc.select-tenants-with-issuer=true #<1>
+quarkus.oidc.hr.application-type=service #<2>
+----
+<1> Tenant will be resolved based on the expected issuer `iss` claim value.
+<2> Only OIDC tenants with the application type 'service' or 'hybrid' are going to be considered during the tenant resolution.
+
```

Comment:
I'd like to expand on this section quite a bit (how the issuer is managed, etc), I can push an update a bit later if you are OK with it, as I'm not sure yet what I can suggest directly here.

Reply:
> I'd like to expand on this section quite a bit (how the issuer is managed, etc), I can push an update a bit later if you are OK with it, as I'm not sure yet what I can suggest directly here.

Please go ahead. I didn't see anything important missing. You have the order and you have the issuer. I am looking forward to see your update.

Reply:
@michalvavrik I've expanded the docs a bit, hope they look ok to you, and will merge once the build passes

Reply:
looks good @sberyozkin